warehouse/templates/manage/account.html

```diff
@@ -364,6 +364,12 @@ <h2 class="sub-title">Two factor authentication (2FA)</h2>
 
   <hr>
 
+  <section id="api-keys">
+    <a href="{{ request.route_path('manage.account.token') }}">Manage API tokens</a>
```

Comment:
Should this be prefaced with `{% if email.verified %}`?

Reply:
Hmm, possibly. Users are currently able to add tokens even without a verified email, but the underlying policy *should* boot them out when they try to use them.

If we do add this, we'll probably need to do it in the forms themselves (to prevent users from navigating directly to the view).

Reply:
@woodruffw Yes - when I tried to use the token with a no-verified-email user account, I did get stopped, with a useful error message! :+1: 

@nlhkabu I'll defer to you here since this feels like a UX issue. My assumption is that, if a token's unusable until/unless the user has a verified email address, then we should stop a no-verified-email user from creating a token in the first place, thus avoiding an unpleasant surprise down the line.

Reply:
Cool -- good to know the underlying policy is working 😄 

I'm on board with restricting token creation to only users with verified emails -- it definitely is an unpleasant surprise to be able to create unusable tokens.